runtime/compiler/runtime/JITClientSession.cpp

```diff
@@ -756,16 +754,13 @@ ClientSessionData::cacheWellKnownClassChainOffsets(unsigned int includedClasses,
    OMR::CriticalSection wellKnownClasses(_wellKnownClassesMonitor);
 
    _wellKnownClasses._includedClasses = includedClasses;
-   memcpy(_wellKnownClasses._classChainOffsets, classChainOffsets,
-          numClasses * sizeof(classChainOffsets[0]));
+   _wellKnownClasses._classChainOffsets = classChainOffsets;
```

Comment:
This changes result in different behavior because `numClasses` is no longer considered.
Please revert this and the formatting changes below.